docs/Growing.md

```diff
@@ -244,6 +244,68 @@ Keep-Alive: timeout=5
 Hello World!
 ```
 
+## Model
+
+In mojo.js we consider web applications simple frontends for existing business logic. That means mojo.js
+is by design entirely _model_ layer agnostic, and you just use whatever JavaScript modules you like most.
```

Comment:
Line length seems sketchy. compared to the next section.

Reply:
I just thought it looked more balanced, but altered first line to use full length now :)